compiler/test/dotty/tools/dotc/transform/TypeTestsCastsTest.scala

```diff
@@ -15,7 +17,7 @@ class TypeTestsCastsTest extends DottyTest:
   @Test def orL = checkFound(List(StringType, LongType), OrType(LongType, StringType, false))
   @Test def orR = checkFound(List(LongType, StringType), OrType(StringType, LongType, false))
 
-  @Test def annot = checkFound(List(StringType, LongType), AnnotatedType(OrType(LongType, StringType, false), Annotation(defn.UncheckedAnnot)))
+  @Test def annot = checkFound(List(StringType, LongType), AnnotatedType(OrType(LongType, StringType, false), Annotation(defn.UncheckedAnnot, NoSpan)))
```

Comment:
Maybe use `Span(0)` as a dummy span. This way we won't have a nonexistent position in any place in the compiler or tests.